### PR TITLE
Reinstate the flight-desktop-restapi restart.sh script

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
The issue preventing the restart has been resolved by closing the file descriptor:
https://github.com/openflighthpc/flight-desktop-restapi/pull/40

The PR has been merged as I needed the tag to build off and it was relatively small. Apparently I fixed some but not all of the dependabot issues.

### CODE REVIEW

File: builders/flight-desktop-restapi/opt/flight/etc/service/types/desktop-restapi/restart.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 OLD_PID="$1"
 
-source "$DIR"/stop.sh "$OLD_PID"
+bash "$DIR"/stop.sh "$OLD_PID"
```
Comment (benarmston) [inline builders/flight-desktop-restapi/opt/flight/etc/service/types/desktop-restapi/restart.sh:32]: Why the change from `source` to `bash`?

Comment (WilliamMcCumstie) [reply builders/flight-desktop-restapi/opt/flight/etc/service/types/desktop-restapi/restart.sh:32]: There has been some uncertainty in general whether the `restart` scripts should be use `source` or `bash`.

However it's possible that the `stop.sh` script might modify various environment variables or use the various bash built ins (`set`, `exit`, etc.). As such it's probably best to execute the script in a sub-shell to protect the parent.

**Comment by WilliamMcCumstie** [comment]
I'm going to merge this PR so other work can be rebased of it
